### PR TITLE
Use exec() and print() functions in both Python 2 and 3 

### PR DESCRIPTION
**Pull request by cclauss**
Also, old style exceptions --> new style

### CODE REVIEW

**Comment by theInfectedDrake** [comment]
This might be one of the greatest fixes of the shitty code I casually wrote all way through. This addition was priceless. 💜

Since this is a big fix, it might take some 3-4 days to review the full code before I can merge it in.

Thank you so much!

File: modules/0x01-OSINT+Footprinting/0x01-PassiveReconnaissance/getconinfo.py [hunk 1]
```diff
@@ -9,69 +9,70 @@
...
+	print(R+'     D O M A I N   C O N T A C T   I N F O')
+	print(R+'    =======================================\n')
 	time.sleep(0.6)
-	print GR+' [*] Importing API Key...'
+	print(GR+' [*] Importing API Key...')
 	try:
 		from files.API_KEYS import FULLCONTACT_API_KEY
-	except IOError, ImportError:
-		print R+' [-] Error while importing key...'
+	except IOError as ImportError:
```
Comment (theInfectedDrake) [inline modules/0x01-OSINT+Footprinting/0x01-PassiveReconnaissance/getconinfo.py]: Something seriously went wrong here. :(

Comment (cclauss) [reply modules/0x01-OSINT+Footprinting/0x01-PassiveReconnaissance/getconinfo.py]: Good catch!  The ambitiousness of the old style syntax is is why there was the move to new style.

File: lib/scapy/build/lib.linux-x86_64-2.7/scapy/tools/check_asdis.py [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 import getopt
 
 def usage():
-    print >>sys.stderr,"""Usage: check_asdis -i <pcap_file> [-o <wrong_packets.pcap>]
+    print("""Usage: check_asdis -i <pcap_file> [-o <wrong_packets.pcap>]
     -v   increase verbosity
     -d   hexdiff packets that differ
     -z   compress output pcap
-    -a   open pcap file in append mode"""
+    -a   open pcap file in append mode""", file=sys.stderr)
```
Comment (theInfectedDrake) [inline lib/scapy/build/lib.linux-x86_64-2.7/scapy/tools/check_asdis.py]: I sense something went wrong here for sure.

File: lib/scapy/build/lib.linux-x86_64-2.7/scapy/tools/check_asdis.py [hunk 3]
```diff
@@ -39,8 +40,8 @@ def main(argv):
         if PCAP_IN is None:
             raise getopt.GetoptError("Missing pcap file (-i)")
     
-    except getopt.GetoptError,e:
-        print >>sys.stderr,"ERROR: %s" % e
+    except getopt.GetoptError as e:
+        print("ERROR: %s" % e, file=sys.stderr)
```
Comment (theInfectedDrake) [inline lib/scapy/build/lib.linux-x86_64-2.7/scapy/tools/check_asdis.py]: Another one.

Comment (theInfectedDrake) [reply lib/scapy/build/lib.linux-x86_64-2.7/scapy/tools/check_asdis.py]: Oh I got it what you're trying to do. Resolved.

**Pull request merged**